tasks/cat2.yml

```
@@ -481,6 +506,35 @@
       - V-38484
       - sshd
 
+- name: "MEDIUM | V-51875 | AUDIT | The operating system, upon successful logon, must display to the user the date and time of the last logon or access via ssh."
+  shell: "grep 'pam_lastlog.so' /etc/pam.d/system-auth"
+  ignore_errors: yes
```

change to `failed_when: no` instead of ignore errors